internal/maintenance/periodic_job_enqueuer.go

```diff
@@ -427,14 +428,16 @@ func (s *PeriodicJobEnqueuer) insertParamsFromConstructor(ctx context.Context, c
 	return insertParams, uniqueOpts, true
 }
 
+const periodicJobEnqueurVeryLongDuration = 24 * time.Hour
```

Comment:
Small typo in the name of this long constant:

```suggestion
const periodicJobEnqueuerVeryLongDuration = 24 * time.Hour
```

Reply:
Oops, thanks. Fixed.